dev-requirements.txt

```diff
@@ -1,5 +1,5 @@
--r requirements.txt
```

Comment:
`requirements.txt` is no longer relevant to dev or are we taking a different approach?

Reply:
`requirements.txt` has been removed in favor of `setup.cfg:options:install_requires`. `tox` packages and installs `apiron` in each virtual environment, so `requests` gets picked up as part of that process. You could optionally `pip install [-e] .` manually if you're doing interactive testing.